lib/LineFollower/ReadyState.cpp

```diff
@@ -97,17 +98,26 @@ void ReadyState::process(StateMachine& sm)
         int16_t         position     = lineSensors.readLine();
         const uint16_t* sensorValues = lineSensors.getSensorValues();
 
+        char msg[Logging::MESSAGE_BUFFER_SIZE] = "";
+        char tmp[10U] = {0};
+
         /* Print line sensor value on console for debug purposes. */
         for (index = 0; index < lineSensors.getNumLineSensors(); ++index)
         {
+            
             if (0 < index)
             {
-                Serial.print(" / ");
+                strcat(msg, " / ");
             }
-            Serial.print(sensorValues[index]);
+            
+            snprintf(tmp, 10U, "%u", sensorValues[index]);

```

Comment:
Using a 10 as buffer size directly is not good. If the buffer size changes, it will be forgotten for sure.